templates/power-off-vm.json

```diff
@@ -346,4 +379,4 @@
         }
     },
     "supportedResources": ["Microsoft.Compute/virtualMachines"]
-}
+}
```

Comment:
Can you have a newline at the end of this file?

Reply:
alright on it